lib/tasks/demo-build.js

```diff
@@ -90,6 +90,7 @@ function buildJs(buildConfig) {
 }
 
 function loadRemoteDemoData(url) {
+	const genericErrorMessage = 'Could not load remote demo data.';
```

Comment:
Could we include the url in the error message please, that way the user might be able to understand why we could not load remote demo data

Reply:
Ignore this, it looks like we add it later on 👌